cmd/trace-agent/config/config.go

```diff
@@ -249,6 +249,23 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 	if k := "apm_config.max_payload_size"; coreconfig.Datadog.IsSet(k) {
 		c.MaxRequestBytes = coreconfig.Datadog.GetInt64(k)
 	}
+	if coreconfig.Datadog.IsSet("apm_config.trace_buffer") {
+		c.TraceBuffer = coreconfig.Datadog.GetInt("apm_config.trace_buffer")
+	}
```

Comment:
I may remove this, or at least keep it undocumented.

I haven't been able to get it to improve submission latency, throughput, or anything with simulated bursty traffic or any other kind of load.

There *might* be use-cases for it, but it is kind of a footgun.